src/js/lib/create-classname-string.js

```diff
@@ -0,0 +1,9 @@
+export default (...classes) =>
```

Comment:
Also shouldn't the naming convention be createClassNameString or createClassnameString instead of create-classname-string?